doc/source/user_guide/indexing.rst

```diff
@@ -1553,9 +1553,7 @@ Index objects
 -------------
 
 The pandas :class:`~pandas.Index` class and its subclasses can be viewed as
-implementing an *ordered multiset*. Duplicates are allowed. However, if you try
-to convert an :class:`~pandas.Index` object with duplicate entries into a
-``set``, an exception will be raised.
+implementing an *ordered multiset*. Duplicates are allowed.
```

Comment:
Why did this change?

Reply:
@mroeschke sorry, this is for https://github.com/pandas-dev/pandas/issues/46440. I thought the comment with number in the parenthesis like (#....) would be sufficient for pointing to a specific issue but it must not. I do not know how to move this PR... :( would you help me here?

Reply:
You can just revert this change like

```
git revert b1521ea
git push origin <your_branch>
```

Reply:
@moreschke do you mean [b4f97fe](https://github.com/pandas-dev/pandas/pull/46711/commits/b4f97feacc4821a711d01a1f5aa58a2490a81cc2)? And how do I PR to a specific issue?

Reply:
oh i got it https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue